src/collections/Users/access/byGlobalRoleOrTenantRoleAssignmentOrDomain.ts

```diff
@@ -70,8 +70,8 @@ export const byGlobalRoleOrTenantRoleAssignmentOrDomain: (method: ruleMethod) =>
       }
     }
 
-    // allow users to read their own record
-    if (args?.id === args.req.user.id) {
+    // allow users to read, update their own record
+    if (args?.id === args.req.user.id && method !== 'delete' && method !== 'create') {
```

Comment:
Very ⛏️ I would write this as
```suggestion
    if (args?.id === args.req.user.id && (method === 'read' || method === 'update')) {
```
But what you have is fine with the comment!

Reply:
I was thinking about how `method` can also be `*`.

_But_, I don't think that really applies here since the `accessBy...` function always uses CRUD actions. 

So I think we'd be safe to explicitly allow read and update and ignore the `*` case. I'm on board, I'll change this.